app/views/host_shelters/properties/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Create an Host Shelter</h1>
+<%= form_for [:host_shelters, @property] do |f| %>
+  <%= f.label :Street_Address %>
+  <%= f.text_field :street_address %>
+
+  <%= f.label :City%>
+  <%= f.text_field :city %>
+
+  <%= f.label :State%>
+  <%= select_tag :state, options_for_select(us_states), include_blank: true %>
+
+  <%= f.label :Rooms_Available%>
+  <%= number_field(:property, :rooms_available, in: 1.0..10.0, step: 1) %><br />
```

Comment:
anyone have good ideas for this choosing rooms_available. 
Do we need to consume pictures too?? ---> Not hard just the uploading part will be minor-ly difficult. 
This form and its styling is not UX friendly/inviting IMO, does anyone else agree??